package.json

```diff
@@ -43,7 +43,8 @@
     "generateDocs": "jsdoc -c ./jsdoc_conf.json",
     "fmt": "standard 'src/**/.js' --fix",
     "lint": "standard 'src/**/*.js'",
-    "clean": "rm -rf ./dist ./node_modules",
+    "clean": "rm -rf ./dist",
```

Comment:
I would rather stay with a single clean script that does it all. If people are using clean as it is, this could break their expectations.